### ISSUE TITLE
PlainReporter colouring output of forbidden-import

### ISSUE THREAD

**Issue by IskyN** [pyta-uoft/pyta#147]
When PlainReporter reports a `forbidden-import` error, it adds colorama colouring though it's not supposed to, resulting in colouring tokens littering plain text files. 
E.g.: running `check_all(reporter=PlainReporter)` on the following code:
```python
import sys
pass
```
results in the following plain-text output (snippet):
```
. . .
    [Line 1] You may not import any modules - you imported \x1b[34msys\x1b[0m on line 1.
. . .
```
(Note the extraneous `\x1b[34m` and `\x1b[0m`.)
The problem seems to be irrelevant of the module being imported.

**Comment by IskyN** [comment]
Update: It seems that the [`forbidden_import_checker`](https://github.com/pyta-uoft/pyta/blob/master/python_ta/checkers/forbidden_import_checker.py#L15) is causing the improper ASCII colour token in the error message. (It illegally imports `colorama` and colours the message text before sending it to the reporter.) 
Not fixing the issue right now because it will be solved along with the addition of the new ROP-made coloured PyTA messages (aka the new `ColorReporter`).

**Comment by IskyN** [comment]
Update: now fixed with commit 5a9f221.
